src/System.Memory/tests/Span/SequenceCompareTo.int.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void ZeroLengthSequenceCompareTo_Int()
```

Comment:
We already have some T = int tests in this file:
https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/SequenceCompareTo.T.cs#L12

Can you please consolidate these (i.e. remove the redundant int tests from SequenceCompare.T.cs?